pages/news/_slug.vue

```diff
@@ -0,0 +1,28 @@
+<template>
```

Comment:
is there a reason for the `_` in the name?

Reply:
It makes it a dynamic page.
https://nuxtjs.org/docs/2.x/directory-structure/pages#dynamic-pages